cfgov/jinja2/v1/_includes/templates/nav/primary-nav.html

```diff
@@ -45,15 +37,7 @@
         {%- endfor %}
     </ul><!-- END .primary-nav_top-level-list -->
 
-    {% do global_dict.update(
-        {
-            'global_eyebrow': {
-                'is_in_list': true
-            }
-
-        })
-    %}
-    {% set value = global_dict.global_eyebrow %}
-    {% include 'molecules/global-eyebrow.html' with context %}
+    {% from 'molecules/global-eyebrow.html' import render as eyebrow with context %}
+    {{ eyebrow() }}
```

Comment:
To following existing conventions this should be `global_eyebrow.render()` so the function call name isn't dependent on the context in which it is imported.